sunpy/net/vso/vso.py

```diff
@@ -920,4 +903,50 @@ def unknown_method(self, response):
 
     @classmethod
     def _can_handle_query(cls, *query):
+        # VSO Queries must have time
+        if not core_attrs.Time in [type(a) for a in query]:
+            return False
         return all([x.__class__.__name__ in core_attrs.__all__ + attrs.__all__ for x in query])
+
+    @classmethod
+    def _attrs_module(cls):
+        return 'vso', 'sunpy.net.vso.attrs'
+
+    @classmethod
+    def register_values(cls):
+        try:
+            return {cls: cls.get_vso_values()}
+        except Exception as e:
+            warnings.warn(f'Fetching VSO attrs failed with {e}')
+            return {cls: {}}
+
+    def get_vso_values():
+        """
+        Reads the VSO keywords and returns a dict with them for the attrs we support.
+        """
+        from sunpy.net import attrs as a
+
+        # Keywords we are after
+        keywords = ["+detector", "+instrument", "+source", "+provider", "+physobs"]
```

Comment:
Doing the query for `+level` also works btw it returns:
```
[{},{"level":"1"},{"level":"1.5"},{"level":"3"},{"level":"2"}]
```

Reply:
Added.